docker/Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN apt install -y build-essential mingw-w64 gcc-aarch64-linux-gnu gcc-arm-linux
 RUN curl https://sh.rustup.rs -sSf | bash -s -- --default-toolchain none -y
 ENV PATH="/root/.cargo/bin:${PATH}"
 
-RUN rustup toolchain install 1.69.0
+RUN rustup toolchain install stable
```

Comment:
i posit that dependencies should be specific numbers instead of automatically updating to latest upstream supply chain

Reply:
I disagree.   We should pick up security patches as soon as they're available, and be forced to deal with their implications.